projects/plugins/boost/changelog/e2e-better_scoping

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+
```

Comment:
I think something went wrong when creating the changelog entry, it's empty.

Reply:
For patch/other changes the comment is not mandatory and I went full on "next next" in cli prompts to create the PR faster.

Reply:
Updated with [977639a](https://github.com/Automattic/jetpack/pull/28913/commits/977639ab09fe21b66e96830cf8bf88614c578250)